Package.swift

```diff
@@ -3,13 +3,10 @@ import PackageDescription
 
 let package = Package(
   name: "Lottie",
-  // TODO: Fix SPM support for macOS, tvOS, and watchOS.
-  // https://github.com/airbnb/lottie-ios/issues/1361
-  // platforms: [.iOS("9.0"), .macOS("10.10"), tvOS("9.0"), .watchOS("2.0")],
-  platforms: [.iOS(.v9)],
+  platforms: [.iOS("9.0"), .macOS("10.10"), .tvOS("9.0")],
```

Comment:
Perhaps as a follow-up, what do you think about starting a `CHANGELOG` in the same style as the one we have for Epoxy? Might be helpful to start capturing some of these

Reply:
Good idea, although I'm not totally sure how we'd want to reference the existing releases (maybe something like "releases 3.2.3 and earlier are tracked [here](https://github.com/airbnb/lottie-ios/releases)")?

Just curious, what do you think are the benefits of having a `CHANGELOG.md` over, say, having release notes on the GitHub release tab (which seems to be how Lottie has handled release notes so far).